boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpuapp.dts

```diff
@@ -28,7 +28,7 @@
 		zephyr,bt-hci = &bt_hci_ipc0;
 		nordic,802154-spinel-ipc = &ipc0;
 		zephyr,canbus = &can120;
-		zephyr,entropy = &prng;
+		zephyr,entropy = &psa_rng;
```

Comment:
Just a suggestion and not a blocker but you can even just leave that as it was with the prng. I removed that before because I was trying to remove the prng all together. But now the prng cannot be removed since there are more platforms using it, so this can stay as it was before if you want.